File: Convert string to camel case.py
text = "The_stealth-warrior"

def to_camel_case(text):
    text = list(text)
    output = []

    for i in range(len(text)):

        if text[i].isalpha():

            if text[i - 1].isalpha():
                output.append(text[i])

            else:
                output.append(text[i].upper())

    return "".join(output)

print(to_camel_case(text))

File: JS/Convert string to camel case.js
let str = "The_stealth-warrior"

function toCamelCase(str){
    let strList = str.split('')
    let output = [];
    for (let i = 0; i < strList.length; i++) {
        if (/[A-Za-z]/.test(strList[i])) {
            if (/[A-Za-z]/.test(strList[i - 1])) {
                output.push(strList[i]);
            } 
            else {
                output.push(strList[i].toUpperCase());
            }
        }
    }
    return output.join('');
}

console.log(toCamelCase(str))